src/FHDAlogger.py

```diff
@@ -15,10 +16,15 @@ def initiateLogger(origin, level):
     '''
 
     try:
-        logging.basicConfig(filename = '../log/' + 
+        curr_path_splitted = str(Path.cwd()).split('/')
```

Comment:
what if you have 3 levels?

Reply:
we may want to use an env file to define absolute path to `src` folder and `log` folder

Reply:
Yes, must of time it may need to be under `/var/log`.
If not define, should just output to stdout